RadarSDK/Radar.m

```diff
@@ -584,13 +586,14 @@ + (void)startTripWithOptions:(RadarTripOptions *)tripOptions
                                                      [RadarSettings removePreviousTrackingOptions];
                                                  }
 
-                                                 if (trackingOptions) {
+                                                 if (trackingOptions && trackingOptions.startTrackingAfter == nil) {
                                                      [self startTrackingWithOptions:trackingOptions];
-                                                 } else if (!Radar.isTracking) {
+                                                 } else if (trackingOptions) {
+                                                     [RadarSettings setTrackingOptions:trackingOptions];
+                                                 } else if (!Radar.isTracking && tripOptions && tripOptions.startTracking) {
```

Comment:
we are only calling `[RadarSettings setPreviousTrackingOptions:[RadarSettings trackingOptions]];` if we are tracking, I think we need to update `previousTrackingOptions` regardless under this implementation to avoid some edge casey situations where we dont' restore our old tracking options.

Reply:
talked in person, we're good on this b/c `previousTrackingOptions` is exclusively about the tracking options prior to the trip lifecycle.